packages/suite/src/views/wallet/staking/components/StakingDashboard/components/EmptyStakingCard.tsx

```diff
@@ -30,7 +30,6 @@ export const EmptyStakingCard = () => {
     const { isStakingDisabled, stakingMessageContent } = useMessageSystemStaking();
 
     const ethApy = useSelector(state => selectPoolStatsApyData(state, account?.symbol));
```

Comment:
pls rename to apy or smthng like that, if there can be solapy as well

Reply:
done, https://github.com/trezor/trezor-suite/pull/16027/commits/0c3a403434ad3f3f1c13ccfde96f287d679b6557